docs/README.md

```diff
@@ -1,17 +1,3 @@
-<ul>
-  <li><a href="#marked">About</a></li>
-  <li><a href="#demo">Demo</a></li>
-  <li><a href="#installation">Installation</a></li>
-  <li><a href="#usage">Usage</a></li>
-  <li><a href="#specifications">Supported Markdown specifications</a></li>
-  <li><a href="#security">Security</a></li>
-  <li><a href="#contributing">Contributing</a></li>
-  <li><a href="#authors">Authors</a></li>
-  <li><a href="#license">License</a></li>
-</ul>
```

Comment:
We don't need navigation in each individual file since we have a global navigation menu now.